src/coreclr/scripts/superpmi.py

```diff
@@ -1859,7 +1879,7 @@ async def create_replay_artifacts(print_prefix, item, self, mch_file, env_vars,
                         # as the LoadLibrary path will be relative to the current directory.
                         with ChangeDir(self.coreclr_args.core_root):
 
-                            async def create_one_artifact(jit_path: str, location: str) -> str:
+                            async def create_one_artifact(jit_path: str, location: str, flags: list[str]) -> str:
```

Comment:
Note that `flags: list[str]` won't work with Python3.8 or lower, list was changed to be generic in 3.9.
It is probably not a big deal that some narrow parts need a higher python version than the minimal supported for the repo.

Reply:
We do run collections using this script on Helix machines which have 3.5 (I think), and merge/upload on the "build" machines which have something different (3.6?), so it could be a problem. But this is replay, which is only done on local user machines.